src/js/components/alert.js

```diff
@@ -22,17 +24,18 @@ class Alert {
 
   setupAlert () {
     const { textMessage, textButton, size, triggerClose } = this.options
-    this.$element = $(this.buildHtml(textMessage, textButton))
 
-    this.modal = $(this.$element).modal({ triggerClose, size }).data('modal')
+    this.element = domParser(this.buildHtml(textMessage, textButton))
```

Comment:
Poderia mover o `this.element` pro construtor? só pra manter a mesma forma que os outros componentes